src/g42-main.cpp

```diff
@@ -304,6 +305,15 @@ int main(int argc, const char **argv) { // the main() function
 	vector<string> argt; // args (without exec name)
 	for (int i=1; i<argc; ++i) argt.push_back(argv[i]);
 	bool early_debug = contains_value(argt, "--d");
+	std::setlocale(LC_ALL, "en_US.UTF-8");
+
+	std::call_once(c_curl_ptr::s_curl_init_flag, []{
+		CURLcode res = curl_global_init(CURL_GLOBAL_DEFAULT);
+		if(res != CURLE_OK) {
+			throw std::runtime_error("curl_global_init error");
+		}
```

Comment:
Needs to be in try-catch, or just print error and exit.

Either way, here or in catch block use: 
std::cerr<<"Error: lib curl init."<<std::endl;